test/tools/format_tool_test.dart

```diff
@@ -37,6 +38,73 @@ void main() {
 
       expect(argParser.options['formatter-args'].type, OptionType.single);
     });
+
+    group('getInputs', () {
```

Comment:
It looks like these tests are all new, but I just moved it from below where it was named differently. The only new test is the followLinks one.